workLoggingSystem.html

```diff
@@ -333,7 +334,8 @@
                     </div>
                     <div class="row formComponent">
                         <div class="col"></div>
-                        <div class="col-md-2" id="createNewTaskSubmit">
+                        <div class="col-md-3" id="createNewTaskSubmit" style="bottom: 40px;">
+                            <div class="col-md-12" style="color: red;visibility: hidden;" id="err">*Please enter the valid details</div>
```

Comment:
Inappropriate bootstrapping and and does not much of meaning, Please use proper bootstrapping technique